### PR TITLE
SE-847-Update-the-Deploy-Test-scripts

### PR DESCRIPTION
**Pull request by ShaneDuncan602**
…ocker process

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -19,7 +19,7 @@ APPLICATION_ERC721_URI_1=nfturigoeshere
 ERC721_PRICING_CONTRACT=0x0000000000000000000000000000000000000000
 ERC20_PRICING_CONTRACT=0x0000000000000000000000000000000000000000
 APPLICATION_AMM_HANDLER=0x0000000000000000000000000000000000000000 
-APPLICATION_ERC721_HANDLER=0x0000000000000000000000000000000000000000
+APPLICATION_ERC721_HANDLER=0xb6c21a92c128230ab6be696ec2a5ec7f2c094f78
```
Comment (Palmerg4) [inline .env]: Is this kept intentionally?

File: docs/userGuides/deployment/DEPLOYMENT-DIRECTORY.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 | Version | Environment     | Contract Name        | Contract Address |
 | :--- |:----------------|:---------------------| :--- | 
-| v1.2.1 | Polygon Amoy    | RuleProcessorDiamond | 0x8D0CcE081d8cb97C68061077883897C4797467E8
+| v1.2.1 | Polygon Amoy    | RuleProcessorDiamond | 0x18693c9efb90c7a00f80d58ca0aa78fb0514dd81
```
Comment (Palmerg4) [inline docs/userGuides/deployment/DEPLOYMENT-DIRECTORY.md]: Are these still v1.2.1?

File: .env (hunk 1 above)
Comment (ShaneDuncan602) [reply .env]: Good catch. That should not have been committed. I don't understand why our .gitignore is not working for this.

File: docs/userGuides/deployment/DEPLOYMENT-DIRECTORY.md (hunk 2 above)
Comment (ShaneDuncan602) [reply docs/userGuides/deployment/DEPLOYMENT-DIRECTORY.md]: Nope, should be 1.3.0

**Review by Palmerg4 — APPROVED** [review]
Looks great!

**Review by mpetersoCode55 — APPROVED** [review]
Great work!

**Pull request merged**